packages/graphql-dynamodb-transformer/src/definitions.ts

```diff
@@ -526,16 +526,19 @@ export function makeSubscriptionField(fieldName: string, returnTypeName: string,
     )
 }
 
+export type typeName = 'Composite' | string;
```

Comment:
Types should by PascalCased, and since it is an export ```typeName``` is too general, I'd suggest ```SortKeyFieldInfoTypeName```, that shows well what the type is used for.